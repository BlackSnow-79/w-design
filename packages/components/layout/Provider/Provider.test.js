"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
// This component is based on styled-components, unnecessary to do overtest
var React = require("react");
var enzyme_1 = require("enzyme");
require("jest-styled-components");
var Provider_1 = require("./Provider");
describe("<Provider/>", function () {
    // Render test
    it("should renders correctly", function () {
        var wrapper = enzyme_1.shallow(React.createElement(Provider_1.default, null));
        expect(wrapper).toMatchSnapshot();
    });
});
//# sourceMappingURL=Provider.test.js.map