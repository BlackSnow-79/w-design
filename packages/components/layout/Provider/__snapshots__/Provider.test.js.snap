// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Provider/> should renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <W.Provider
    theme={Object {}}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Root />,
      "theme": Object {
        "breakpoints": Array [
          "24em",
          "32em",
          "48em",
          "64em",
          "80em",
        ],
        "colors": Object {
          "B10": "#3ECBFF",
          "B20": "#00B7FA",
          "B30": "#00A7E4",
          "B40": "#009AD3",
          "B5": "#B1EAFF",
          "B50": "#008CC0",
          "B60": "#007099",
          "B7": "#7CDCFF",
          "B70": "#004660",
          "BLACK": "#000",
          "G10": "#79F2C0",
          "G20": "#57D9A3",
          "G30": "#36B37E",
          "G40": "#006644",
          "G5": "#E2FFEE",
          "G50": "#00875A",
          "G7": "#ABF5D1",
          "N1": "#FBFBFB",
          "N10": "#6C869B",
          "N2": "#EFF3F6",
          "N20": "#60798E",
          "N3": "#E2EAF2",
          "N30": "#566E82",
          "N4": "#CED9E3",
          "N40": "#4D6171",
          "N5": "#BECEDC",
          "N50": "#394B59",
          "N6": "#9DB3C6",
          "N60": "#293742",
          "N7": "#90A9BF",
          "N70": "#202B33",
          "N8": "#87A0B5",
          "N80": "#182026",
          "N9": "#768FA4",
          "N90": "#10161A",
          "O10": "#FFE380",
          "O20": "#FFC60A",
          "O30": "#FA9B00",
          "O40": "#EF8100",
          "O5": "#FFF7D7",
          "O50": "#E15B00",
          "O7": "#FFF19F",
          "P10": "#998DD9",
          "P20": "#8777D9",
          "P30": "#6554C0",
          "P40": "#5243AA",
          "P5": "#EAE6FF",
          "P50": "#403294",
          "P7": "#C0B6F2",
          "R10": "#FF5252",
          "R20": "#F22828",
          "R30": "#DC1F1F",
          "R40": "#C61515",
          "R5": "#FFADAD",
          "R50": "#B40404",
          "R7": "#FF8383",
          "T10": "#79E2F2",
          "T20": "#00C7E6",
          "T30": "#00B8D9",
          "T40": "#008DA6",
          "T5": "#E6FCFF",
          "T50": "#00A3BF",
          "T7": "#B3F5FF",
          "WHITE": "#FFF",
        },
        "fontSizes": Array [
          12,
          14,
          16,
          20,
          24,
          32,
          48,
          64,
          72,
          96,
        ],
        "fontWeights": Object {
          "bold": 700,
          "normal": 400,
          "xbold": 900,
        },
        "fonts": Object {
          "0": "system-ui, sans-serif",
          "base": "Roboto, \\"Helvetica Neue\\"",
          "sans": "-apple-system, BlinkMacSystemFont, system-ui, sans-serif",
        },
        "letterSpacings": Object {
          "caps": "0.25em",
          "normal": "normal",
        },
        "lineHeights": Array [
          1,
          1.125,
          1.25,
          1.5,
        ],
        "space": Array [
          0,
          4,
          8,
          16,
          32,
          64,
          128,
        ],
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Root />,
        "theme": Object {
          "breakpoints": Array [
            "24em",
            "32em",
            "48em",
            "64em",
            "80em",
          ],
          "colors": Object {
            "B10": "#3ECBFF",
            "B20": "#00B7FA",
            "B30": "#00A7E4",
            "B40": "#009AD3",
            "B5": "#B1EAFF",
            "B50": "#008CC0",
            "B60": "#007099",
            "B7": "#7CDCFF",
            "B70": "#004660",
            "BLACK": "#000",
            "G10": "#79F2C0",
            "G20": "#57D9A3",
            "G30": "#36B37E",
            "G40": "#006644",
            "G5": "#E2FFEE",
            "G50": "#00875A",
            "G7": "#ABF5D1",
            "N1": "#FBFBFB",
            "N10": "#6C869B",
            "N2": "#EFF3F6",
            "N20": "#60798E",
            "N3": "#E2EAF2",
            "N30": "#566E82",
            "N4": "#CED9E3",
            "N40": "#4D6171",
            "N5": "#BECEDC",
            "N50": "#394B59",
            "N6": "#9DB3C6",
            "N60": "#293742",
            "N7": "#90A9BF",
            "N70": "#202B33",
            "N8": "#87A0B5",
            "N80": "#182026",
            "N9": "#768FA4",
            "N90": "#10161A",
            "O10": "#FFE380",
            "O20": "#FFC60A",
            "O30": "#FA9B00",
            "O40": "#EF8100",
            "O5": "#FFF7D7",
            "O50": "#E15B00",
            "O7": "#FFF19F",
            "P10": "#998DD9",
            "P20": "#8777D9",
            "P30": "#6554C0",
            "P40": "#5243AA",
            "P5": "#EAE6FF",
            "P50": "#403294",
            "P7": "#C0B6F2",
            "R10": "#FF5252",
            "R20": "#F22828",
            "R30": "#DC1F1F",
            "R40": "#C61515",
            "R5": "#FFADAD",
            "R50": "#B40404",
            "R7": "#FF8383",
            "T10": "#79E2F2",
            "T20": "#00C7E6",
            "T30": "#00B8D9",
            "T40": "#008DA6",
            "T5": "#E6FCFF",
            "T50": "#00A3BF",
            "T7": "#B3F5FF",
            "WHITE": "#FFF",
          },
          "fontSizes": Array [
            12,
            14,
            16,
            20,
            24,
            32,
            48,
            64,
            72,
            96,
          ],
          "fontWeights": Object {
            "bold": 700,
            "normal": 400,
            "xbold": 900,
          },
          "fonts": Object {
            "0": "system-ui, sans-serif",
            "base": "Roboto, \\"Helvetica Neue\\"",
            "sans": "-apple-system, BlinkMacSystemFont, system-ui, sans-serif",
          },
          "letterSpacings": Object {
            "caps": "0.25em",
            "normal": "normal",
          },
          "lineHeights": Array [
            1,
            1.125,
            1.25,
            1.5,
          ],
          "space": Array [
            0,
            4,
            8,
            16,
            32,
            64,
            128,
          ],
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
